variablesview.py

```diff
@@ -57,6 +58,7 @@ def val(self):
         return repr(self.value)
 
     def text(self):
+        #return "%s = {%s} %s" % (self.name, self.type_name(), self.val())
         return "%s = {%s} %s" % (self.name, self.type_name(), self.val())
```

Comment:
While we're at it, let's modernise this code and remove the old % formatting!

```suggestion
        return "{} = {{{}}} {}".format(self.name, self.type_name(), self.val())
```